playwright/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM mcr.microsoft.com/playwright:v1.18.1-focal-amd64
```

Comment:
I wonder, does this still work on your machine if, instead of ending with `-amd64`, you do `-${BUILDARCH}`? The work to alter the other Dockerfiles and etc. to make this cross-platform hasn't landed yet, but, if that pull the correct thing on your machine, I'd be super interested. Either way, no pressure to tweak: will run this on an M1 and play after merged.

Reply:
Oh, and we should ask @bensteinberg to mirror the relevant images up to our registry and switch this line to point there to avoid Docker Hub's limits on pulling the image.

Reply:
I don't think it's necessary here, since we're pulling from Microsoft, not Docker Hub.